src/components/Link/Link.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { Link } from './Link'
+
+export default {
+  title: 'Link',
+  parameters: {
+    info: `Link component
+
+    Based on Typography guide: https://designsystem.digital.gov/components/typography/#links`,
+  },
+}
+
+export const Default = (): React.ReactElement => (
+  <p>
+    <Link href={'#'}> This</Link> is a text link.
+  </p>
+)
+
+/*
+ * TODO: sort out how to test visited links in storybook
+ *
+ * export const Visited = (): React.ReactElement => (
```

Comment:
Looking at the USWDS source (https://designsystem.digital.gov/components/typography/#links) I think they just added the `usa-color-text-visited` class to their docs theme for demonstration purposes. We probably could do the same in a Storybook-only CSS file (these can be added in `./storybook`):

```
.usa-color-text-visited {
  color: color($theme-link-visited-color);
}
```

Reply:
we may also want to do something similar to the `Default` story so that it _never_ gets the `:visited` style applied. IMO adding CSS to Storybook to enforce the intended styles for demonstration purposes is a 💯   valid thing to do.

Reply:
@suzubara does https://github.com/trussworks/react-uswds/pull/309/commits/813c393bbe7471bd0e0b0b1b8e4b3a1b1f4093df workable for this??

Reply:
yeah that looks great!